lib/client.js

```diff
@@ -42,6 +50,27 @@ module.exports = function (dependencies) {
         });
       }
     }, this.config.heartBeat).unref();
+    this.manageBroadcastHealthCheckInterval = setInterval(() => {
```

Comment:
We now have two `sessions` on the client:

1. The original session which is used for standard notifications along with broadcast notifications
2. The other session is the `manageBroadcast` session, which I will rename to `manageChannel` session soon